ODataConnectedService/src/Templates/ODataT4CodeGenerator.cs

```diff
@@ -6808,8 +6808,8 @@ internal override void WriteCastToMethods(string baseTypeName, string derivedTyp
 this.Write(this.ToStringHelper.ToStringWithCulture(derivedTypeFullName));
 
 this.Write("\r\n        \'\'\' </summary>\r\n        \'\'\' <param name=\"source\">source entity</param>\r" +
-        "\n        <Global.System.Runtime.CompilerServices.Extension()>\r\n        Public " +
-        "Function CastTo");
+        "\n        <Global.System.Runtime.CompilerServices.Extension()>\r\n        Public Fu" +
```

Comment:
consider splitting the string on a whitespace or new line than in the middle of a word

Reply:
This is autogenerated code